package.json

```diff
@@ -148,6 +148,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "yarn run lint",
+      "pre-push": "yarn test:unit",
```

Comment:
please revert this line, I'll rather push it from my PR, so we won't have conflicts. In my PR I moved this config into a separate file

Reply:
I am not a big fan of forcing tests on pre-push, but other repos in the company fo it too and I kind of got used to it :) WDYT?

Reply:
@mathio My opinion that in general - it is quite useful. However, it starts being annoying when tests take quite a bit of time to run. Since it's not the case - I think there's nothing to worry about (for now) :)

Reply:
Hey @mathio I will complete the review once I can check the demo pages (@prilutskiy is bringing thme up & public again). The code changes look good to me!

Reply:
Demo is deployed here: https://embed.typeform.com/f44bf2f06f4f57d22065e78769012a2006d12797/popup.html

However the mock form did not get deployed. I will need to look into it.

Reply:
Ok, thanks @mathio